### PR TITLE
Add new tagline - The CMS for Creators. Stable. Secure. Instantly Familiar.

### LINKED ISSUE #653: Micro rebranding - The CMS for Creators :: Stable. Secure. Instantly Familiar.
Following up on the marketing meeting via https://forums.classicpress.net/t/marketing-meeting-january-17-2021/2968/9

We are micro rebranding.

**ClassicPress**
**The CMS for Creators**
Stable. Secure. Instantly Familiar.

This needs to be done in a number of files like the 
- [ ] readme file
- [ ] About 
- [ ] etc

Will update this list while fixing the PR.

### PR DESCRIPTION
**Pull request by bahiirwa**
Closes #653

## Description
Chaging the tagline and byline from

```
ClassicPress
A business-focused CMS
Powerful. Versatile. Predictable.
```

 to

```
ClassicPress
The CMS for Creators
Stable. Secure. Instantly Familiar.
```

## How has this been tested?
No tests to write as these are changes to texts. 

Clarity can be made on `tests/phpunit/data/admin/features.json` file. Do we need to rephrase the wording? Many instances of this appear like below.

Screenshot 2021-01-23 at 19 00 35

### Before
Screenshot 2021-01-23 at 18 57 19

Screenshot 2021-01-23 at 18 58 15

### After
Screenshot 2021-01-23 at 18 54 11

Screenshot 2021-01-23 at 18 56 29

## Types of changes
- Branding texts

### CODE REVIEW

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -26,11 +26,11 @@
 		<p class="about-text">
 			<?php printf(
 				/* translators: link to "business-focused CMS" article */
-				__( 'Thank you for using ClassicPress, the <a href="%s">business-focused CMS</a>.' ),
+				__( 'Thank you for using ClassicPress, the <a href="%s">CMS for Creators</a>.' ),
 				'https://www.classicpress.net/blog/2018/10/29/classicpress-for-business-professional-organization-websites/'
```
Comment (nylen) [inline src/wp-admin/about.php:30]: This link needs an update. I think we can set it to https://link.classicpress.net/the-cms-for-creators and have that redirect to https://www.classicpress.net for now.

Comment (bahiirwa) [reply src/wp-admin/about.php]: Recommendation noted.

**Comment by timbocode** [comment]
The wording in `src/readme.html` also needs some attention. 

Also, in the same file:

- there are also two references to an outdated **releases page** link which is now dead. This should probably be changed to https://link.classicpress.net/releases which is then redirected to https://forums.classicpress.net/c/announcements/release-notes/48 for now.
- the petitions link needs to be updated.

**Comment by bahiirwa** [comment]
Giving this a refresh to get checks to pass.

**Comment by nylen** [comment]
Something went wrong with the refresh - there are now a bunch of other commits included here.

It looks like 2291a57, af0a5c4, and 0f539f0 are the commits that should be in this PR. Something like this should work to get things back on track:

```
# Update the code from https://github.com/ClassicPress/ClassicPress
# (this should be set as the `upstream` remote)
git fetch upstream

# Reset the `rebrand` branch to the latest development code
git checkout upstream/develop -B rebrand

# Re-apply the commits that should go into this PR
git cherry-pick 2291a57
git cherry-pick af0a5c4
git cherry-pick 0f539f0

# Refresh the PR
# (your `bahiirwa/ClassicPress` fork should be set as the `origin` remote)
git push origin rebrand -f
```

**Comment by bahiirwa** [comment]
Thank you. This is committed.

File: src/readme.html [hunk 2]
```diff
@@ -10,17 +10,17 @@
...
+<p>Our purpose is to provide a stable, secure, and instantly familiar content management system driven by the needs of the community. We invite you to connect with us and <a href="#resources">add your voice to the conversation.</a>.</p>
+<p style="text-align: right">&#8212; The ClassicPress Directors</p>
 
 <h2 id="installation">Installation</h2>
 <p>For more information about installing ClassicPress, see our <a href="https://docs.classicpress.net/installing-classicpress/">installation documentation</a>.</p>
 <ol>
-	<li>Download the latest release of ClassicPress from our <a href="https://www.classicpress.net/releases/">releases page</a>.</li>
+	<li>Download the latest release of ClassicPress from our <a href="https://link.classicpress.net/releases/">releases page</a>.</li>
```
Comment (nylen) [inline src/readme.html:23]: Need to fix this URL at https://github.com/ClassicPress/subdomain-link first.

File: src/readme.html [hunk 3]
```diff
@@ -80,12 +80,12 @@ <h2 id="resources">Online Resources</h2>
...
 	<dt><a href="https://www.classicpress.net/blog/">ClassicPress Blog</a></dt>
-		<dd>This is where you&#8217;ll find the latest updates and news related to ClassicPress. Recent ClassicPress news appears in your administrative dashboard by default.</dd>
+		<dd>This is where you&#8217;ll find the latest updates and news related to ClassicPress as well as tutorials and other interesting snippets of information. Recent ClassicPress news appears in your administrative dashboard by default.</dd>
 	<dt><a href="https://forums.classicpress.net/c/support">ClassicPress Support Forums</a></dt>
 		<dd>The support forums are a great way to get help from other ClassicPress users. Many community members and ClassicPress leaders are active here.</dd>
 	<dt><a href="https://www.classicpress.net/join-slack/">ClassicPress Slack</a></dt>
 		<dd>There is an active online chat channel that is used for discussion among people who use ClassicPress and plan its further development.</dd>
-	<dt><a href="https://petitions.classicpress.net/">ClassicPress Petitions</a></dt>
+	<dt><a href="https://forums.classicpress.net/c/governance/petitions/77">ClassicPress Petitions</a></dt>
```
Comment (nylen) [inline src/readme.html:88]: Let's add a link to https://github.com/ClassicPress/subdomain-link for this.

File: src/wp-admin/about.php [hunk 4]
```diff
@@ -26,11 +26,11 @@
 		<p class="about-text">
 			<?php printf(
 				/* translators: link to "business-focused CMS" article */
-				__( 'Thank you for using ClassicPress, the <a href="%s">business-focused CMS</a>.' ),
-				'https://www.classicpress.net/blog/2018/10/29/classicpress-for-business-professional-organization-websites/'
+				__( 'Thank you for using ClassicPress, the <a href="%s">CMS for Creators</a>.' ),
+				'https://link.classicpress.net/the-cms-for-creators'
```
Comment (nylen) [inline src/wp-admin/about.php:30]: Need to fix this URL at https://github.com/ClassicPress/subdomain-link first.

Comment (bahiirwa) [reply src/wp-admin/about.php:30]: @striebwj This is the missing puzzle piece.

**Comment by bahiirwa** [comment]
@striebwj This is a textual change. I adding community tag. As I have reviewed it again. 

The only catch is the missing link `https://link.classicpress.net/the-cms-for-creators`

File: src/wp-admin/about.php (hunk 4 above)
Comment (striebwj) [reply src/wp-admin/about.php:30]: Do we have anything we can even point to?

File: src/readme.html (hunk 2 above)
Comment (striebwj) [reply src/readme.html:23]: Resolved by ClassicPress/subdomain-link#3

File: src/readme.html (hunk 3 above)
Comment (striebwj) [reply src/readme.html:88]: Resolved by ClassicPress/subdomain-link#3

File: src/wp-admin/about.php (hunk 4 above)
Comment (bahiirwa) [reply src/wp-admin/about.php:30]: @striebwj That link was what we were supposed to link to. We need an article for that on website.

Comment (striebwj) [reply src/wp-admin/about.php:30]: I see the PR in subdomain was just pointing to the home page. I think we are okay to merge this, and merge ClassicPress/subdomain-link#3 until we get an article up on the site.

**Review by striebwj — APPROVED** [review]

File: src/readme.html (hunk 3 above)
Comment (striebwj) [reply src/readme.html:88]: @bahiirwa do we want to add a link redirect for this?

```suggestion
	<dt><a href="https://link.classicpress.net/petitions">ClassicPress Petitions</a></dt>
```

**Review by mattyrob — COMMENTED** [review]
All looks good to me, we just need the URL working:
https://link.classicpress.net/the-cms-for-creators

**Review by mattyrob — APPROVED** [review]

**Comment by striebwj** [comment]
@nylen and I just deployed the link update, we should be all good here 👍

**Pull request merged**